js/helpers/errors.js

```diff
@@ -0,0 +1,23 @@
+var isUndefined = require( "lodash/isUndefined" );
+
+/**
+ * Shows and error trace of the error message in the console if the console is available.
+ *
+ * @param {string} [errorMessage=""] The error message.
+ */
+function showTrace( errorMessage ) {
+	if ( isUndefined( errorMessage ) ) {
+		errorMessage = "";
```

Comment:
Because I want to be able to call this function without an error message and still put a trace in the console.